app/src/main/java/com/r4/notifications/mirror/MainActivity.java

```diff
@@ -23,32 +23,35 @@
 public class MainActivity extends AppCompatActivity {
     private final static String TAG = "nm.MainActivity";
 
-    public static Context sContext;
+    //public static Context sContext;
     public static NotificationMirror notificationMirror;
 
     private NotificationManagerCompat notificationManager;
     private SharedPreferences shPref;
     private SharedPreferences.Editor editor;
     private Notification testNotification;
 
+    private Context context;
+
```

Comment:
I would advise to change the name of the parameter to something more descriptive like applicationContext